### ISSUE TITLE
unreliable test: "getConfigsMappedToTemplates(type=undefined)"

### ISSUE THREAD

**Issue by JadenSimon** [aws/aws-toolkit-vscode#2398]
```
  1) LaunchConfiguration
       getConfigsMappedToTemplates(type=undefined) returns target=template + target=api resources:

      AssertionError [ERR_ASSERTION]: Expected values to be strictly deep-equal:

0 !== 1
```

Likely a race condition with globals.

**Comment by JadenSimon** [comment]
Here's another test that's likely caused by the same thing:
```
   1) ActivationReloadState
       getSamInitState
         path defined, with runtime:
     AssertionError [ERR_ASSERTION]: Unexpected Launch Path value was retrieved
      at Context.<anonymous> (src/test/shared/activationReloadState.test.ts:165:20)
      at Generator.next (<anonymous>)
      at fulfilled (dist/src/test/shared/activationReloadState.test.js:9:58)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

**Comment by JadenSimon** [comment]
Looks resolved by #2409
